pkgs/development/r-modules/default.nix

```diff
@@ -424,7 +424,6 @@ let
     showtext = [ pkgs.pkgconfig ];
     spate = [ pkgs.pkgconfig ];
     stringi = [ pkgs.pkgconfig ];
-    sys = [ pkgs.libapparmor ];
     sysfonts = [ pkgs.pkgconfig ];
```

Comment:
I don't much about our R infrastructure. Are these overrides only for a particular version of the module or do they affect multiple?

Reply:
We don't have more than one version of any given R package in Nixpkgs.

Generally speaking, those overrides affect the (one) package with that particular attribute name.